apps/blog/src/posts/transitions-demo/transitions-demo.md

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: Mimic Native Transitions In Your Progressive Web App
+excerpt: 
+description: 
+date: 2023-10-04
+updatedDate: 2023-10-04
+trending: true
+featured: true
+image: 
+isPost: true
+backUrl: '/'
+author:
+  name: Jaylyn Barbee
+  twitter: 
+  title: PWABuilder Engineer
+tags:
+  - PWA
+  - PWA Starter
+---
+
+Good native applications take advantage of subtle transitions from page to page that give life to the application and achieving this just became easier for web applications. 
+
+Previously, a combination of Typescript and CSS was required to replicate transitions like that of a native app, but thanks to the “CSS View Transitions” API that just became a lot easier. It’s as simple as calling .startViewTransition() and you’ve already implemented a cross fade transition from one page to another. 
+
+Choosing when and where to use these transitions in your PWA is arguably more important than using them in the first place. Implementing a consistent and accessible set of transitions across your PWA can give your app that native feel but using transitions out of place can ruin the feel of your app all together. 
+
+This blog will cover some transition tricks that to help upgrade your Progressive Web App and give it a more native feel.
+
+## Types of Transition
+
+Each type of transition has an optimal situation where it is best to use them, so before deciding which transition to use, it might be best to understand what you are trying to achieve by using the transition in the first place.  
```

Comment:
I realized this is a pattern for @Jaylyn-Barbee. A lot of your paragraphs are one sentence. But really they should be a couple of sentences. Hope this is not too nitpicking...
```suggestion
Each type of transition has an optimal situation where it is best to use them. So, before deciding which transition to use, it might be best to understand what you are trying to achieve by using the transition in the first place.  
```